src/main/java/de/samply/share/client/control/ConfigurationBean.java

```diff
@@ -144,8 +147,23 @@ public void setConfigurationTimingsMap(
     this.configurationTimingsMap = configurationTimingsMap;
   }
 
+  /**
+   * Get configuration enum as boolean.
+   *
+   * @param configuration configuration enum.
+   * @return configuration enum as boolean.
+   */
   public boolean getAsBoolean(EnumConfiguration configuration) {
     return ConfigurationUtil.getConfigurationElementValueAsBoolean(configuration);
   }
 
+  /**
+   * Save automatically form information after any change.
+   *
+   * @param event event after modifying any form field.
+   */
+  public void retune(AjaxBehaviorEvent event) { // Note: the argument is optional.
```

Comment:
please change the method name
It doesnt give any information what this method is doing